xbmc/filesystem/BlurayDirectory.cpp

```diff
@@ -212,6 +215,42 @@ void CBlurayDirectory::GetRoot(CFileItemList &items)
     items.Add(item);
 }
 
+void CBlurayDirectory::FindMountPoint(std::string* file)
```

Comment:
This is specific to Linux. Wouldn't it be possible to move to the base directory and somehow implement it only on Linux? Other implementations would return file